azure-pipelines.yml

```diff
@@ -108,27 +108,25 @@ stages:
         sudo rm -rf ../*.deb
       displayName: "Cleanup"
 
-    - task: DownloadPipelineArtifact@2
-      inputs:
-        source: specific
-        project: build
-        pipeline: 9
-        artifact: sonic-swss-common.amd64.ubuntu20_04
-        runVersion: 'latestFromBranch'
-        runBranch: 'refs/heads/master'
-      displayName: "Download sonic swss common deb packages"
-
     - task: DownloadPipelineArtifact@2
       inputs:
         artifact: sonic-buildimage.vs
       displayName: "Download sonic-buildimage.vs artifact"
 
     - script: |
-        set -x
         sudo apt-get update
-        sudo apt-get install libyang0.16 -y
-        sudo dpkg -i --force-confask,confnew ../libswsscommon_1.0.0_amd64.deb
-        sudo dpkg -i ../python3-swsscommon_1.0.0_amd64.deb
+        sudo apt-get install -y make libtool m4 autoconf dh-exec debhelper cmake pkg-config \
+                         libhiredis-dev libnl-3-dev libnl-genl-3-dev libnl-route-3-dev libnl-nf-3-dev swig3.0 \
+                         libpython2.7-dev libboost-dev libboost-serialization-dev uuid-dev libzmq5 libzmq3-dev  python3-pip \
+                         cmake libgtest-dev libgmock-dev libyang-dev
+        sudo pip3 install pytest
+        cd src/sonic-swss-common
+        ./autogen.sh
+        dpkg-buildpackage -rfakeroot -us -uc -b -j$(nproc)
+        sudo dpkg -i ../libswsscommon_*.deb ../python3-swsscommon_*.deb
```

Comment:
>-i [](http://example.com/codeflow?start=18&length=2)

Why dropping the options `--force-confask,confnew` ?

Reply:
Will add it back, need to keep the options, thanks, Qi.